index.html

```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```

Comment:
Recommend modifying ", refresh to play again"  to ", refresh <F5> to play again, hit <ESC> key to exit game, <F12> to log out.